### ISSUE TITLE
Make it possible to edit suggestions

### ISSUE THREAD

**Issue by Suvitruf** [g3rv4/Traducir#107]
Right now you can only delete current suggestion and add new one. Would be nice to add edit option for suggestions.

**Comment by g3rv4** [comment]
\o/ feel free to work on this issue if you want to! it could be 100% frontend (where an edit could delete the old suggestion and create a new one) or you could add an edit suggestion method on the API.

If you go with the latter, then make sure you add a `SuggestionHistory` entry so that we can tell what happened ;)

**Comment by topcatarg** [comment]
Edit would be weird only at front end.. there is no EDIT at the back. It should be the second idea. Anyway. this is just a new reason, a button on the front to tell is an edit, (edit my last suggestion, only if open) and store new suggestion, and reverse the last one to the history (also deleting it, but not for a delete reason).

**Comment by g3rv4** [comment]
I don't think it would be weird to tackle this only as a frontend change. This is exactly what's happening right now.

I have suggestion A that I want to edit, then I:

1. Delete suggestion A (a suggestionHistory is created to show the deletion)
2. Create suggestion B

If the frontend did those 2 things, everything would be kept consistent with what we're doing right now.

If we add the edit action, then we would add a new SuggestionHistoryType, and on the SuggestionHistory we should store the old value (so that we can keep track of what happened).

Personally, I don't mind either option ;) so whatever the person that codes this decides works for me.

**Comment by topcatarg** [comment]
ahhh.. ok.. I was thinking to write an edit in the history.. but a delete and create is fine with me!

**Comment by g3rv4** [comment]
Makes sense, I'm sold on the idea that the backend approach is better ;)
